integrations/postgresql/src/main/kotlin/com/xebia/functional/xef/vectorstores/postgresql/postgres.kt

```diff
@@ -23,7 +23,7 @@ val createMemoryTable: String =
        conversation_id TEXT NOT NULL,
        role TEXT NOT NULL,
        content TEXT UNIQUE NOT NULL,
-       timestamp TIMESTAMP NOT NULL,
+       timestamp BIGINT NOT NULL
```

Comment:
Here I'm changing the `xef_memories` schema from using `TIMESTAMP` to `BIGINT`. I thought that it wasn't very important for this value to be a `TIMESTAMP`, we just needed something that could be ordered. Maybe for performance issues it's better to have this as a `TIMESTAMP` instead of a `BIGINT`. However, having it as a `TIMESTAMP` here and modelling it in the application as a `Long` required translating back and forth, which was more difficult. Let me know your opinion on that and if you agree I can change it and implement both translations